### PR TITLE
Add a section about sso external collaborators

### PR DESCRIPTION
**Pull request by Kakulukian**

### CODE REVIEW

**Comment by Pierrci** [comment]
LGTM, can we just update the screenshots just above to reflect the new setting too?

File: docs/hub/security-sso.md [hunk 1]
```diff
@@ -72,3 +72,13 @@ You must map at least one admin role.
...
+
+
+#### External Collaborators
+
+This enables certain users within your organization to access resources without completing the Single Sign-On (SSO) flow described before. This can be helpful when you work with external parties who aren't part of your organization's Identity Provider (IdP) but require access to specific resources.
+
+To add a user as an "External Collaborator" visit the `SSO/Users Management` section in your organization's settings. Once added, these users won't need to go through the SSO process. 
+
+However, they will still be subject to your organization's access controls ([Resource Groups](./security-resource-groups)). 
+It's crucial to manage their access carefully to maintain your organization's data security.
```
Comment (julien-c) [inline docs/hub/security-sso.md:84]: this last line is from HuggingChat no?

**Review by julien-c — APPROVED** [review]

File: docs/hub/security-sso.md (hunk 1 above)
Comment (Kakulukian) [reply docs/hub/security-sso.md:84]: No Grammarly, it has rephrased important to crucial

Comment (Pierrci) [reply docs/hub/security-sso.md:84]: 😁

**Comment by Pierrci** [comment]
> LGTM, can we just update the screenshots just above to reflect the new setting too?

still the old ones it seems 🥲

**Pull request merged**